app/controllers/autocomplete_controller.rb

```diff
@@ -0,0 +1,33 @@
+class AutocompleteController < ApplicationController
+  def github_repos
+    @repos = search_github_repos(params[:query])
+
+    respond_to do |format|
+      format.html { render partial: 'autocomplete/repository_suggestions', locals: { repos: @repos } }
+    end
+  end
+
+  private
+
+  def required_scopes
```

Comment:
How does this behave if we don't have the right scopes? Since we're requesting this endpoint via ajax, there's no opportunity to ask for more scopes.